macos/FluentUI/Button.swift

```diff
@@ -193,6 +193,10 @@ open class Button: NSButton {
 
 	private var hoverBackgroundColor: NSColor {
 		if #available(macOS 10.14, *) {
+			if (fillColor == .clear)
```

Comment:
Should this be in the #available(macOS 10.14, *) block? The code being added doesn't use anything 10.14 specific